CMakeLists.txt

```diff
@@ -65,6 +65,7 @@ option( TRACCC_BUILD_ALPAKA "Build the Alpaka sources included in traccc"
    FALSE )
 option( TRACCC_BUILD_IO "Build the IO module (needed by examples, performance, testing)" TRUE )
 option( TRACCC_BUILD_TESTING "Build the (unit) tests of traccc" TRUE )
+option( TRACCC_BUILD_BENCHMARK "Build the benchmarks of traccc" TRUE )
```

Comment:
Super subjective comment, but I myself would call it `TRACCC_BUILD_BENCHMARKS` (plural) instead. :thinking:

Reply:
IT was BENCHMARKS at the beginning but I changed to BENCHMARK to follow the detray standard...

Reply:
No actually I was referring other variables :p will fix it to BENCHMARKS